### DISCUSSION TITLE
The cargo scooter recipe is missing FPGA Circuit

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14556] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Underwater Scooter contain FPGA in recipe but cargo scooter have not.
Was this how it was intended?
Is the cargo scooter recipe missing FPGA Circuit?

### Reproduction steps

look at the recipes Underwater Scooter and cargo scooter and be logic.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next patch after the upcoming hotfix. Closing discussion.
